spec/services/markdown_processor/parser_spec.rb

```diff
@@ -158,6 +158,12 @@ def generate_and_parse_markdown(raw_markdown)
       test = generate_and_parse_markdown(code_span)
       expect(test).to eq("<p><a href=\"#chapter-1\">Chapter 1</a></p>\n\n")
     end
+
+    it "does not render CSS classes" do
+      expect(generate_and_parse_markdown("<center class=\"w-100\"></center>"))
```

Comment:
Nitpick (nonblocking): I find escaped quotes in strings hard to grok

```suggestion
      expect(generate_and_parse_markdown(%w{<center class="w-100"></center>}))
```

Reply:
Did you include the `w` on purpose, because that would turn this into an array with one string. I think you meant `%{}`, no?

Reply:
hahahaha Yes, I absolutely did 🤦🏼‍♂️